### ISSUE TITLE
Action Item Button Title Length

### ISSUE THREAD

**Issue by jsoape** [BuildFire/sdk#246]
I am fairly new to js programming, but I am making some small modifications to one of your existing plugins to better suit our application. However, I am unfamiliar with where or how to affect the length limit on the action items button title field on the content side- http://d.pr/i/ATd5. Any help would be greatly appreciated.

**Comment by DanielHindi** [comment]
Unfortunatly, You are unable to at this point. There is a reason for the limit. Even if it was extended all you'd see would be an ellipsis at the end
"My Title is Lon..."

**Comment by jsoape** [comment]
I seem to only be able to title with about 12 characters. However, when we were using the original plug in without any modifications we could title it with up to maybe 18 or so. How did it change between the two?

**Comment by DanielHindi** [comment]
Thats a good point let me check with the developers. I'll get back to you soon

**Comment by jsoape** [comment]
Much appreciated. Ill look for your response!

**Comment by DanielHindi** [comment]
Its now queued up to get fixed. Thanks for pointing that out

**Comment by jsoape** [comment]
One other issue we are facing... the only modifications we made to the plugin was on the item file. We adjusted the options menu to the bottom of the content area and made it into a button, and changed the titles on a few areas. That was all we did. But the images and titles are not loading on the folder side now (http://d.pr/i/5kQq). Images and everything look great when you actually get into the item itself. Any help would be appreciated. When we first modified it everything was functioning it seemed, then last night the images wouldnt load. I am attaching the file we modified and some screen shots. 
[seminarNotesPlugin-master 2.zip](https://github.com/BuildFire/sdk/files/434827/seminarNotesPlugin-master.2.zip)
